tardis/energy_input/GXPacket.py

```diff
@@ -31,7 +31,8 @@ class GXPacketStatus(IntEnum):
     ("nu_cmf", float64),
     ("status", int64),
     ("shell", int64),
-    ("time_current", float64),
+    ("time_start", float64),
```

Comment:
I don't think this is a good name for a variable that is updated once the packet has travelled a distance.